react-native-aztec/android/src/main/java/org/wordpress/mobile/ReactNativeAztec/ReactAztecTextShadowNode.java

```diff
@@ -10,6 +10,13 @@
     private @Nullable ReadableMap mTextMap = null;
     private @Nullable Integer mColor = null;
 
+    @Override
+    public void setDefaultPadding(int spacingType, float padding) {
+        // Do nothing in order to avoid adding unwanted default padding to views
+        // (https://github.com/wordpress-mobile/gutenberg-mobile/issues/992) due to
+        // https://github.com/facebook/react-native/blob/6ebd3b046e5b71130281f1a7dbe7220eff95d74a/ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputShadowNode.java#L70
+    }
+
```

Comment:
Another approach that works equally well is to explicitly set padding to 0 on the `ShadowNode` when it is constructed instead of preventing the default padding from being set like I am doing:

```
    public ReactAztecTextShadowNode() {
        super();
        setPadding(Spacing.START, 0);
        setPadding(Spacing.TOP, 0);
        setPadding(Spacing.END, 0);
        setPadding(Spacing.BOTTOM,0);
    }
```

I do not have a strong preference between these two approaches. It is nice how ignoring the default padding does not have us setting a padding of 0 both here and in the `ReactAztecManager`.